frontend/appflowy_flutter/pubspec.lock

```diff
@@ -1559,10 +1559,10 @@ packages:
     dependency: transitive
     description:
       name: platform
-      sha256: "9b71283fc13df574056616011fb138fd3b793ea47cc509c189a6c3fa5f8a1a65"
+      sha256: "12220bb4b65720483f8fa9450b4332347737cf8213dd2840d8b2c823e47243ec"
       url: "https://pub.dev"
     source: hosted
-    version: "3.1.5"
+    version: "3.1.4"
```

Comment:
I noticed that the versions in pubspec.lock have been changing frequently recently.

The versions of these three packages changed when running pub get. (not sure why)

Reply:
What flutter version are you all on?

I'm on `3.22.1` - could be related

Reply:
I'm using Flutter 3.22.2.

```
Flutter 3.22.2 • channel [user-branch] • unknown source
```

Reply:
3.22.3 for me. It could be from the newest stable 3.24.x though